### ISSUE TITLE
I want to know how to train this net

### ISSUE THREAD

**Issue by Breakmydream** [torrvision/siamfc-tf#20]

**Comment by mingchuan123** [comment]
Me too.

**Comment by bertinetto** [comment]
This version uses a matconvnet-pretrained version.
You can train new models using the code from the conv5 baseline of CFNet: [](https://github.com/bertinetto/cfnet/blob/master/src/training/run_experiment_baseline_conv5.m)

However, since the model is fully specified in this TensorFlow project, it shouldn't be too much work to add a logistic loss and the training code.

If you can wait though, we might come up with a PyTorch (better) version at some point next year.

**Comment by ahkarami** [comment]
+1 for PyTorch Code

**Comment by omkar13** [comment]
+1 for PyTorch code

**Comment by bertinetto** [comment]
I have actually finished with the pytorch version a couple of weeks ago but it's a bit behind the original implementation performance.

I might try and release it, perhaps people can help levelling the performance?

**Comment by ahkarami** [comment]
Yes, if you release the PyTorch version maybe we can help you to increase its performance.

**Comment by ahkarami** [comment]
Dear @bertinetto,
Have you released the PyTorch version of the project?

**Comment by ssulun16** [comment]
Dear Luca,
Can you release the PyTorch version? I'd be happy to work on it.
